setup.py

```diff
@@ -149,4 +149,5 @@ def get_long_description():
               },
           zip_safe=False,
           test_suite='selftests',
-          python_requires='>=2.6')
+          python_requires='>=2.6',
+          install_requires=['stevedore'])
```

Comment:
OK after discussion with @clebergnu we agree that this is an improvement and we don't want it to cover everything (especially the python 2.6 with many custom dependencies).

As we don't handle 2.6 we should not specify 2.7 specific stevedore version so let's just keep it as is (without any specific version required) and leave it on python 2.6 users to use requirements.

The ImportErrors mentioned by @lmr  are expected as plugins are expected to warn about malfunctioning, but the core functionality is fine.